docs/2.0/manual.md

```diff
@@ -428,7 +429,9 @@ The `input phred offset` and `output phred offset` are applicable only to [FASTQ
 {: .example}
 
 # Leading Segment
-Pheniqs constructs new segments for the output read and must generate corresponding identifiers and replicate metadata from the input to the output segments. Since segments can potentially disagree on metadata, one input segment is elected as the leader and used as a template when constructing the output segments. The leading segment property is a reference to an [input segment index](glossary.html#input_segment) and defaults to **0** if omitted.
+Pheniqs constructs new segments for the output read and must generate corresponding identifiers and replicate metadata from the input. Since segments can potentially disagree on metadata, one input segment is elected as the leader and used as a template when constructing the output segments. The leading segment property is a reference to an [input segment index](glossary.html#input_segment) and defaults to **0** if omitted.
+
+**EXAMPLE OF WHEN THIS WOULD BE USEFUL**
```

Comment:
please remove the comments so I can actually merge this. Any examples added would be welcome :)

Reply:
I would provide an example but I don't remember the scenario that you explained, so this on is on you.